corehq/apps/reports/static/reports/js/reports.async.js

```diff
@@ -240,12 +235,10 @@ hqDefine("reports/js/reports.async", function () {
         });
 
         self.loadingIssueModal.on('hide hide.bs.modal', function () {
-            if (self.issueAttempts > 0) {
-                self.hqLoading = $(self.loaderClass);
-                self.hqLoading.find('.js-loading-spinner').addClass('hide');
-                self.hqLoading.find('h4').text(gettext('We were unsuccessful loading the report:'))
-                    .attr('style', 'margin-bottom: 10px;');
-            }
+            self.hqLoading = $(self.loaderClass);
+            self.hqLoading.find('.js-loading-spinner').addClass('hide');
+            self.hqLoading.find('h4').text(gettext('We were unsuccessful loading the report:'))
```

Comment:
```suggestion
            self.hqLoading.find('h4').text(gettext('We were unsuccessful loading the report:'));
```

Reply:
hmm I wonder why stickler isn't picking up on these kinds of things anymore

Reply:
whoops that's because this isn't the end of the line

Reply:
ahhhhh